api/src/test/java/org/openmrs/api/PatientServiceTest.java

```diff
@@ -3406,4 +3408,33 @@ public void mergePatients_shouldNotFailIfMultiplePatientsHaveActiveOrderOfDiffer
 	public void processDeath_shouldThrowAPIExceptionIfPatientIsNull() throws Exception{
 		patientService.processDeath(null, new Date(), new Concept(), "unknown");
 	}
+
+	/**
+	 * @verifies move all the obs with same hierarchy for the patient after merge.
+	 * @see PatientService#mergePatients(org.openmrs.Patient, org.openmrs.Patient)
+	 */
+	@Test
+	@Verifies(value = "Should move all the obs with same hierarchy for the patient after merge.", method = "mergePatient(Patient)")
```

Comment:
Yeah, I was confused between both, I saw few tests using `@Verifies` annotation, and few javadoc style. That's why I used both, I will delete one of them. I will do the change you mentioned in first comment.